python/dgl/graphbolt/utils/internal.py

```diff
@@ -35,10 +35,10 @@ def save_data(data, path, fmt):
         raise RuntimeError(f"Unsupported format: {fmt}")
 
     # Perform necessary conversion.
-    if fmt == "numpy" and isinstance(element, torch.Tensor):
-        element = element.cpu().numpy()
-    elif fmt == "torch" and isinstance(element, np.ndarray):
-        element = torch.from_numpy(element).cpu()
+    if fmt == "numpy" and isinstance(data, torch.Tensor):
```

Comment:
such typo of variable name is not captured by unit test. what a shock! could you add a dedicated unit test for this API?

Reply:
@keli-wen should do it in #6449 .

Reply:
Get